src/Platform/Microsoft.Testing.Platform.MSBuild/buildMultiTargeting/Microsoft.Testing.Platform.MSBuild.targets

```diff
@@ -352,8 +352,16 @@
       This let's terminal logger know that it should show `Testing` in the output,
       and not the additional internal details, such as target names.
     -->
+    <!-- Passing UseAppHost, IsExecutable, TargetDir, AssemblyName, and _NativeExecutableExtensions is mostly workaround -->
+    <!-- until we implement https://github.com/microsoft/testfx/issues/5091 properly -->
+    <!-- For now, we construct the correct "RunCommand" via these properties for a specific scenario that's known to be broken -->
     <CallTarget Targets="_TestRunStart" />
     <InvokeTestingPlatformTask TargetPath="$(TargetPath)"
+                               UseAppHost="$(UseAppHost)"
+                               IsExecutable="$(_IsExecutable)"
+                               TargetDir="$(TargetDir)"
+                               AssemblyName="$(AssemblyName)"
+                               NativeExecutableExtension="$(_NativeExecutableExtension)"
```

Comment:
@baronfel Could you please confirm if it's okay to use `_NativeExecutableExtension` and `_IsExecutable`? We want to avoid using things that may potentially break in future.

Reply:
Both are set by the .NET SDK in props files and so should be available. They _are_ _-prefixed properties, which means we don't like to encourage their direct use, but I don't think we make this data available in any other way at this time. cc @dsplaisted for heads-up that more folks are taking data dependencies on SDK-private members.